catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/SourceProcessingDetailsImpl.java

```diff
@@ -36,6 +38,28 @@ public SourceProcessingDetailsImpl(List<String> warnings) {
     this.warnings = warnings;
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Braces should be added to conform to google style as listed on the "Coding Standards" page on confluence. This statement applies to the entire PR.

>4.1.1 Braces are used where optional
>Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.